AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5018] SG2 | Code standards to-dos for Resource page

## Ticket(s)

**Jira Ticket**
- [EWL-5018: SG2 | Code standards to-dos for Resource page](https://issues.ama-assn.org/browse/EWL-5018)

## Description
As a developer using the Resource page pattern in SG2, the pattern should be built consistently with Twig and utilize the jQuery UI Tabs library.

This update refactors relevant twig templates, json, js, and scss for resource page tab functionality.

## To Test

- Pull `feature/EWL-5018-code-standards-resource-page` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Page > Resource](http://localhost:3000/?p=pages-resource).
- Confirm page still works as it did in [Style Guide 2](https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-resource)


## Visual Regressions

Updated reference images for resource page backstop tests have been included in this PR.

## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
